packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -264,6 +268,12 @@ struct WindowData {
         SendMessageW(cmbEngines, (UINT)CB_ADDSTRING, (WPARAM)0, (LPARAM)TEXT("V8"));
         // SendMessageW(cmbEngines, CB_SETCURSEL, (WPARAM) static_cast<int32_t>(self->m_jsEngine), (LPARAM)0);
 
+        auto cmbTheme = GetDlgItem(hwnd, IDC_THEME);
+        SendMessageW(cmbTheme, (UINT)CB_ADDSTRING, (WPARAM)0, (LPARAM)TEXT("Default"));
+        SendMessageW(cmbTheme, (UINT)CB_ADDSTRING, (WPARAM)0, (LPARAM)TEXT("Light"));
+        SendMessageW(cmbTheme, (UINT)CB_ADDSTRING, (WPARAM)0, (LPARAM)TEXT("Dark"));
```

Comment:
don't need the casts since these are literals. Also TEXT is deprecated (we build everything in UTF16 nowadays)
```suggestion
        SendMessageW(cmbTheme, CB_ADDSTRING, 0, (LPARAM)L"Default");
        SendMessageW(cmbTheme, CB_ADDSTRING, 0, (LPARAM)L"Light");
        SendMessageW(cmbTheme, CB_ADDSTRING, 0, (LPARAM)L"Dark");
```

Reply:
I was copying the code a few lines above. Someone may want to clean that up too.